src/views/RandomView.vue

```diff
@@ -1,9 +1,28 @@
+<script setup>
+import { ref } from "vue"
+  import BoosterCard from "../components/BoosterCard.vue"
+  const booster = ref(null)
+  const getNewFact = () => {
+    fetch("https://uselessfacts.jsph.pl/random.json?language=en")
```

Comment:
Just as a small suggestion, it might be helpful to move this network request into its own file.  Also remember to include some error handling including the `.catch` and a check before parsing the response!